### PR TITLE
Unskip Enterprise CRD tests

### PR DESCRIPTION
**Pull request by lkysow**
These pass now with the latest changes.

### CODE REVIEW

**Comment by lkysow** [comment]
These passed on the last run when only running the controller tests (https://app.circleci.com/pipelines/github/hashicorp/consul-helm/1773/workflows/b4a96a44-1670-417d-ac62-1d80255f497f/jobs/4440)

**Review by thisisnotashwin — APPROVED** [review]
🦊

**Review by ndhanushkodi — APPROVED** [review]
I see the tests pass locally! Just added a comment checking whether we need to continue overriding global.image or not.

File: test/acceptance/tests/controller/controller_namespaces_test.go [hunk 1]
```diff
@@ -78,7 +77,7 @@ func TestControllerNamespaces(t *testing.T) {
 				"connectInject.enabled":         "true",
 
 				// todo: remove when Helm chart updates to 1.8.4
-				"global.image": "hashicorp/consul-enterprise:1.8.4-ent",
+				"global.image": "hashicorp/consul-enterprise:1.9.0-ent-beta1",
```
[the remark this answers is not in the document]
Comment (lkysow) [reply test/acceptance/tests/controller/controller_namespaces_test.go:80]: Oops, yeah that comment is incorrect. The controller tests use the service-intentions CRD which requires 1.9.0.

So really that comment should say:
```
// todo: remove when Helm chart updates to 1.9.0
```

I'll push that change.

[the remark this answers is not in the document]
Comment (ndhanushkodi) [reply test/acceptance/tests/controller/controller_namespaces_test.go:80]: Sweet, thanks!

**Pull request merged**